_topic_map.yml

```diff
@@ -1568,6 +1568,8 @@ Topics:
       File: cnv-installing-qemu-guest-agent
     - Name: Viewing the IP address of NICs on a virtual machine
       File: cnv-viewing-ip-of-vm-nic
+    - Name: Enabling KubeMacPool for Pods and virtual machines 
```

Comment:
On D/S we support KubeMacPool only for VirtualMachines, so we should drop all the Pod mentions from the doc